### ISSUE TITLE
 train data on trained weight file

### ISSUE THREAD

**Issue by encore2020** [pjreddie/darknet#483]
If I have trained data and created the weight file.
Next time I add some new data, could I train it by using the last created weight file?

**Comment by drforester** [comment]
Yes, this is how I trained the detector on a dataset called "nfpa" using the Darknet 448x448 pretrained model (https://pjreddie.com/darknet/imagenet/#darknet19_448).  

~$ ./darknet detector train cfg/nfpa.data cfg/yolo-nfpa.2.0.cfg darknet19_448.conv.23  

You would just replace "darknet19_448.conv.23" with your wieights file to keep training.

**Comment by MizbaMohammed** [comment]
@drforester Hi David,  I want to add 3 classes from COCO dataset to the tiny YOLO which was trained on VOC dataset. what all changes do I have to make in that case? Do I have to combine all the data and train tiny Yolo again? Thank you.

**Comment by ahsan856jalal** [comment]
so total classes = 20 +3=23
so in cfg file classes =23 and filters in the convolution layer before region layer =num*(classes+5)
